### PR TITLE
add the tuf client to the verifier script 

### PR DESCRIPTION
**Pull request by asraa**
* Add a go-tuf client to download the targets if the metadata has been published
* Writes out the downloaded verified targets
* Fixes up minor comment in the README.md

Signed-off-by: Asra Ali <asraa@google.com>

### CODE REVIEW

File: cmd/verify/main.go [hunk 1]
```diff
@@ -167,22 +173,117 @@ func main() {
...
+			if err != nil {
+				log.Printf("error reading trusted TUF root: %s", *tufRoot)
+				return
+			}
+			meta := map[string]json.RawMessage{"root.json": rootMeta}
+			local := tuf.MemoryStore(meta, nil)
+			repo, err := tuf.NewRepo(local)
+			if err != nil {
+				log.Printf("error reading trusted TUF local: %s", *tufRoot)
+				return
```
Comment (SantiagoTorres) [inline cmd/verify/main.go]: Not sure of the context, but would it be useful to return Error/a negative integer so as to make calling processes (and shells) aware that things failed?

**Review by dlorenc — APPROVED** [review]

File: cmd/verify/main.go (hunk 1 above)
Comment (asraa) [reply cmd/verify/main.go]: Yeah, the places where it really errors I exit with 1. I should do that here!
Thank you!

Comment (asraa) [reply cmd/verify/main.go]: done!

**Comment by asraa** [comment]
would love re-approval for manual branch protection :)

**Review by dlorenc — APPROVED** [review]

**Pull request merged**